Makefile.core.mk

```diff
@@ -63,6 +63,8 @@ api_protos := $(shell find $(api_path) -type f -name '*.proto' | sort)
 api_pb_gos := $(api_protos:.proto=.pb.go)
 
 
+.PHONY: default
+.DEFAULT_GOAL :=
```

Comment:
first time I've seen this in a Makefile.  The upstream documentation states: ```
.DEFAULT_GOAL
Sets the default goal to be used if no targets were specified on the command line (see Arguments to Specify the Goals). The .DEFAULT_GOAL variable allows you to discover the current default goal, restart the default goal selection algorithm by clearing its value, or to explicitly set the default goal. The following example illustrates these cases:
```

I don't understand why this is needed.  Could you expand?

Reply:
This is a bad submission. Sorry for the trouble, maybe I should close this pr.